docs/docs/Components/components-vector-stores.md

```diff
@@ -319,6 +319,30 @@ For more information, see the [FAISS documentation](https://faiss.ai/index.html)
 |----------------|------------------------|--------------------------------|
 | vector_store   | FAISS                  | A FAISS vector store instance configured with the specified parameters. |
 
+## Graph RAG
+
+This component performs Graph RAG (Retrieval Augmented Generation) traversal in a vector store, enabling graph-based document retrieval.
+For more information, see the [Graph RAG documentation](https://datastax.github.io/graph-rag/).
+
+For an example flow, see the **Graph RAG** template.
+
+### Inputs
+
+| Name | Display Name | Info |
+|------|--------------|------|
+| embedding_model | Embedding Model | Specify the Embedding Model. This is not required for Astra Vectorize collections. |
+| vector_store | Vector Store Connection | Connection to the vector store. |
+| edge_definition | Edge Definition | Edge definition for the graph traversal. |
```

Comment:
Defining "edge definition" by saying it's the "edge definition" for something is not very helpful.

Reply:
Added a link to [GraphRAG documentation](https://datastax.github.io/graph-rag/reference/graph_retriever/edges/)